### PR TITLE
integration + unittest fixes 

### PR DESCRIPTION
**Pull request by HereThereBeDragons**

### CODE REVIEW

File: test/common/testutil.cc [hunk 1]
```diff
@@ -76,6 +76,30 @@ pid_t GetParentPid(const pid_t pid) {
   return parent_pid;
 }
 
+std::string GetProcessname(const pid_t pid) {
+#ifdef __APPLE__
+  std::err << "Implementation missing to get process name by pid on MAC\n";
```
Comment (vvolkl) [inline test/common/testutil.cc]:
```suggestion
  std::cerr << "Implementation missing to get process name by pid on MAC\n";
```
On macOS this should actually be easy to get using `proc_pidpath` from `libproc.h`, I can provide the implementation in a future PR.

**Review by jblomer — APPROVED** [review]
Good catches!

**Comment by jblomer** [comment]
@cernvm-bot cloudtest

**Comment by HereThereBeDragons** [comment]
@cernvm-bot cloudtest

**Review by jblomer — APPROVED** [review]

**Pull request merged**